packages/pinned_pool/src/pinned_slab.rs

```diff
@@ -539,6 +558,9 @@ impl<'s, T, const CAPACITY: usize> PinnedSlabInserter<'s, T, CAPACITY> {
             ),
         };
 
+        // Disarm the cleanup guard since initialization succeeded.
+        scopeguard::ScopeGuard::into_inner(cleanup_guard);
```

Comment:
Overly verbose name here - repo guidelines say to use `use` statements to import names, this should just be `ScopeGuard::into_inner()`.